keps/sig-api-machinery/1929-built-in-default/README.md

```diff
@@ -276,12 +276,9 @@ endpoint today, but we have to decide what to do with the new default
 values of built-in types. The kube-openapi generated schemas will
 include them.
 
-We proppose to:
-1. leave CRDs not publishing defaults
-2. filter out defaults in the built-in type schemas too in a first step
-3. but possibly reconsider this behaviour in the future when the
-necessity of defaults for client-side merging (e.g. through kustomize)
-and influences on the ecosystem of default mis-use is better understood.
+We propose to:
+1. to allow the CRDs to publish the defaults field which helps the client to set the <zero-value>.
```

Comment:
Actually, it's the opposite. We absolutely don't want to clients to do anything. You should add a paragraph after that list that says something like:
```md
Clients should NOT be modified to use that default for fields that are not specified, and the documentation must make that very clear. The default is informational only, according to OpenAPI documentation: 
> The default value is the one that the server uses if the client does not supply the parameter value in the request.
```

Reply:
Added the mentioned point